lib/dispatcher.lib.php

```diff
@@ -491,7 +491,7 @@ public function removefriend($dispatcher, &$reqData, &$out) {
 
 		$userid = $psdb->escape($curuser['userid']);
 		$player = $psdb->escape($reqData['player']);
-		$res = $psdb->query(
+		if($res = $psdb->query(
```

Comment:
This `if(` appears unclosed by a matching `)`.

Reply:
Ah, thank you. Because I couldn't actually test this code, I wasn't sure if fixing the bug would be if($res = $psdb->query [...]) ']' . $reqData['player'] . ' has been removed from your friend list.'; or what I've changed on the following line, and I guess when I was still looking into which was correct, I forgot to undo the unclosed if(. Seeing as the pull request was closed, I'll instead start looking into a server-side friends list system.